src/components/SearchBar/SearchBar.jsx

```diff
@@ -1,9 +1,9 @@
 import { useEffect, useState } from "react";
 import { getTopics } from "../../utils/util-api-calls";
-import { capitaliseTopic } from "../../utils/util-data-formatters";
-import "./SearchBar.css"
+import { capitaliseFirstLetter } from "../../utils/util-data-formatters";
+import "./SearchBar.css";
 
-function SearchBar({ setSearchParams }) {
+function SearchBar({ searchParams, setSearchParams }) {
```

Comment:
Nice to see you are using useSearchParams to use queries rather than state to manage the sort options 😄